samples/sqs/native-integration/Sqs_6/Receiver/Receiver.csproj

```diff
@@ -1,13 +1,12 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFrameworks>net7.0;net6.0;net48</TargetFrameworks>
     <OutputType>Exe</OutputType>
     <LangVersion>8.0</LangVersion>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Newtonsoft.Json" Version="13.*" />
     <PackageReference Include="NServiceBus" Version="8.*" />
-    <PackageReference Include="NServiceBus.AmazonSQS" Version="6.*" />
+    <PackageReference Include="NServiceBus.AmazonSQS" Version="6.1" />
```

Comment:
This could be a wildcard again once we release right?

Reply:
yes - I have changed it back.